packages/terra-clinical-item-collection/src/ItemCollection.jsx

```diff
@@ -2,123 +2,102 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import ResponsiveElement from 'terra-responsive-element';
 import 'terra-base/lib/baseStyles';
-import './ItemCollection.scss';
 
-import createListView from './_CreateListView';
-import createTableView from './_CreateTableView';
+import Item from './Item';
+import ListView from './_ListView';
+import TableView from './_TableView';
+import './ItemCollection.scss';
 
 const propTypes = {
   /**
-  * The breakpoint to switch from a table view to a list view. Breakpoint options are tiny, small, medium, large, or huge.
-  */
+   * The breakpoint to switch from a table view to a list view. Breakpoint options are tiny, small, medium, large, or huge.
+   */
   breakpoint: PropTypes.oneOf(['tiny', 'small', 'medium', 'large', 'huge']),
   /**
-   * The styles to spread to the table. Table style options are isPadded and isStriped.
+   * The items that will flex between either a list item or a table row.
    */
-  tableStyles: PropTypes.shape({
-    isPadded: PropTypes.bool,
-    isStriped: PropTypes.bool,
+  children: PropTypes.node.isRequired,
+  /**
```

Comment:
TODO: Flatten theses to `hasStartAccessory`, `numberOfDisplays`, `hasComment`, `hasEndAccessory`

@StephenEsser @tbiethman